pkg/agent/controller/egress/egress_controller.go

```diff
@@ -196,6 +207,18 @@ func NewEgressController(
 	if trafficShapingEnabled && !openflow.OVSMetersAreSupported() {
 		klog.Info("EgressTrafficShaping feature gate is enabled, but it is ignored because OVS meters are not supported.")
 	}
+
+	eventBroadcaster := record.NewBroadcaster()
+	eventBroadcaster.StartStructuredLogging(0)
+	runtime.Must(scheme.AddToScheme(k8sscheme.Scheme))
```

Comment:
my previous comment is not addressed

Reply:
It is required to add our custom scheme to the standard scheme so that the event recorder is able to know the type of object involved, otherwise it won't be able to construct reference to the egress object.

Reply:
1. From the recoder's implementation, I think it only needs to know the target object's scheme, which can be found in `scheme`, I don't get why it must call `AddToScheme` to add K8s scheme.
2. Even if it needs to call `AddToScheme`, it should be called in `init` func, not at runtime, check my previous comment: https://github.com/antrea-io/antrea/pull/5765#discussion_r1423673727